rest/context.go

```diff
@@ -72,6 +73,8 @@ type RequestContext interface {
 	// string is returned with the error set.
 	NextURL() (string, error)
 
+	BuildURL(string, string) (string, error)
```

Comment:
Please provide a docstring beginning with the name of the function like the other methods on the interface. I'd also recommend naming the parameters. From this definition I have no idea how to use the function.